aiounifi/models/event.py

```diff
@@ -305,3 +306,39 @@ def version_from(self) -> str:
     def version_to(self) -> str:
         """Version to."""
         return self.raw.get("version_to", "")
+
+
+@dataclass
+class Meta:
+    """Meta description of UniFi websocket data."""
+
+    rc: str
+    message: MessageKey
+    data: dict[str, Any]
+
+    @classmethod
+    def from_dict(cls, data: dict[str, Any]) -> "Meta":
+        """Create meta instance from dict."""
+        return cls(
+            rc=data.get("rc", ""),
+            message=MessageKey(data.get("message", "")),
+            data=data,
+        )
+
+
+@dataclass
+class WebsocketData:
+    """Websocket package representation."""
+
+    meta: Meta
+    event: Event
+    data: dict[str, Any]
+
+    @classmethod
+    def from_dict(cls, data: dict[str, Any]) -> "WebsocketData":
+        """Create data container instance from dict."""
+        return cls(
+            meta=Meta.from_dict(data.get("meta", "")),
+            event=Event(data["data"]),
+            data=data,
```

Comment:
I would only put the data (without meta) in the data field. It is the part that is used from the data field and meta is separately available.
```suggestion
            data=data["data"],
```